t04_refactored.py

```diff
@@ -134,9 +134,52 @@ def team_2_adv(username, delay):
 
 
 def team_3_adv(username, delay):
-    pass
-    # TODO Add your code here
-    return True
+    """
+       we're creating an idea, giving you options to stay alive. can you stay alive? this game allow for you to pick between a good choice and a bad choice.
+           """
+    print("\n")
+    print("You find a room that contains two gold statues.")
+    sleep(delay)
+    statues = input("Do you take the left or right statue, or don't pick up the statue?: ")
+    print("\n")
+
+    if statues == "left":
+        # good choice
+        print("You pick up the statue on the left.")
+        sleep(delay)
+        print("A passageway opens and you see outside.")
+        print("Congrats! You escaped the cave!")
+        sleep(delay)
+    elif statues == "right":
+        # bad choice
+        print("You pick up the statue on the right.")
+        sleep(delay)
+        print("A passageway opens above you.")
+        sleep(delay)
+        print("A bunch of snakes and spiders fall on you!")
+        isDead = False
+        sleep(delay)
+    else:
+        # neutral choice
+        print("You don't pick up either of the statues.")
+        sleep(delay)
+        print("Nothing happens... You are still stuck in the cave.")
+        sleep(delay)
+
+    # TODO Make sure to add the additional check if the user makes the "bad" choice!
+    if isDead == True:
```

Comment:
Your code never sets `isDead` to `True` prior to this line, meaning this code never gets reached.